packages/next-codemod/bin/upgrade.ts

```diff
@@ -412,7 +412,7 @@ function isUsingAppDir(projectPath: string): boolean {
  */
 async function suggestTurbopack(packageJson: any): Promise<void> {
   const devScript: string = packageJson.scripts['dev']
-  if (devScript.includes('--turbopack')) return
+  if (devScript && devScript.includes('--turbo')) return
```

Comment:
```suggestion
  if (devScript?.includes('--turbo')) return
```

nit: How about optional?